module/reducer.js

```diff
@@ -41,7 +45,8 @@ export function createExperiments (experiments) {
 			}
 
 			case 'SET_EXPERIMENT_VARIANT':
-				return Object.assign({}, state, {[action.experiment]: action.variant})
+                return findVariant(experiments[action.experiment], action.variant) ?
```

Comment:
the same here, I'd prefer an if statement

Reply:
ok, added